Profiles/ProfilesAPI.py

```diff
@@ -1,6 +1,64 @@
-from flask import Blueprint
+from flask import Blueprint, make_response, request
 import sys
 from db import db
+from flask_cors import cross_origin
+
 sys.path.append("../")
 
-profiles_api = Blueprint("profiles", __name__)
\ No newline at end of file
+profiles_api = Blueprint("profiles", __name__)
+
+
+@profiles_api.route("/<int:id>", methods=["GET"])
+@cross_origin(supports_credentials=True)
+def get_profile(id):
+    try:
+        data = {'name': db[id]['name'], 'scores': db[id]['scores']}
+        response = {'status': 'success', 'data': data}
+    except Exception as e:
+        return {"err": "An error has occured", "status": " failed"}, 500
+
+    return make_response(response)
+
+
+@profiles_api.route("/profiles", methods=["POST"])
+@cross_origin(supports_credentials=True)
+def create_new_profile():
+    try:
+        data = request.get_json()
+        db.append(data)
+        response = {'status': 'success'}
+    except Exception as e:
+        return {"err": "An error has occured", "status": " failed"}, 500
+
+    return make_response(response)
+
+
+@profiles_api.route("/<int:id>", methods=["DELETE"])
+@cross_origin(supports_credentials=True)
+def delete_profile(id):
+    try:
+        id = int(id)
+        deleted = db[id]
+        new_db = [profile for profile in db if profile != deleted]
```

Comment:
consider using `del db[id]` instead. your code does not actually delete the person from db. 
e.g. if i do a DELETE request to this API, when i delete id 1, your output tells me that you have deleted id 1. when i proceed to delete id 0, your output tells me that you have deleted id 0 but the id 1 person is still in db.